app/javascript/publish/application.js

```diff
@@ -23,8 +23,22 @@ try {
   const providerTemplate = (result) =>
     result && `${result.name} (${result.code})`
 
+  const schoolAutocomplete = document.getElementById('school-autocomplete')
```

Comment:
We may need to clean this up a little we'll need one more of these for the accredited provider look up in upcoming tickets. Something to think about